tools/devtools/utils.mjs

```diff
@@ -0,0 +1,262 @@
+import {readFile, writeFile, mkdir} from 'fs/promises';
+// eslint-disable-next-line node/no-extraneous-import
```

Comment:
Will remove this line if we add `nunjucks` as direct dependecy.

Reply:
Yes, let's add Nunjucks as a direct dependency.